### PR TITLE
add java+yaml to readme

### PR DESCRIPTION
**Pull request by susanev**
pls let me know what i got wrong, thank you!

### CODE REVIEW

**Review by iwahbe — APPROVED** [review]
Do you think we should center the icons? Anyway, LGTM!

**Comment by susanev** [comment]
> Do you think we should center the icons? Anyway, LGTM!

its kind of a pain to do in markdown, and they are all mostly the same size, so gonna leave as is.

File: README.md [hunk 1]
```diff
@@ -189,9 +189,11 @@ The Pulumi CLI v1 and v2 are no longer supported. If you are not yet running v3,
 | -- | -------- | ------ | ------- |
 | <img src="https://www.pulumi.com/logos/tech/logo-js.png" height=38 />     | [JavaScript](./sdk/nodejs) | Stable  | Node.js 12+  |
 | <img src="https://www.pulumi.com/logos/tech/logo-ts.png" height=38 />     | [TypeScript](./sdk/nodejs) | Stable  | Node.js 12+  |
```
Comment (susanev) [inline README.md:191]:
```suggestion
| <img src="https://www.pulumi.com/logos/tech/logo-js.png" height=38 />     | [JavaScript](./sdk/nodejs) | Stable  | Node.js 14+  |
| <img src="https://www.pulumi.com/logos/tech/logo-ts.png" height=38 />     | [TypeScript](./sdk/nodejs) | Stable  | Node.js 14+  |
```

File: README.md [hunk 2]
```diff
@@ -187,11 +187,13 @@ The Pulumi CLI v1 and v2 are no longer supported. If you are not yet running v3,
 
 |    | Language | Status | Runtime |
 | -- | -------- | ------ | ------- |
-| <img src="https://www.pulumi.com/logos/tech/logo-js.png" height=38 />     | [JavaScript](./sdk/nodejs) | Stable  | Node.js 12+  |
-| <img src="https://www.pulumi.com/logos/tech/logo-ts.png" height=38 />     | [TypeScript](./sdk/nodejs) | Stable  | Node.js 12+  |
-| <img src="https://www.pulumi.com/logos/tech/logo-python.png" height=38 /> | [Python](./sdk/python)     | Stable  | Python 3.6+ |
+| <img src="https://www.pulumi.com/logos/tech/logo-js.png" height=38 />     | [JavaScript](./sdk/nodejs) | Stable  | Node.js 14+  |
+| <img src="https://www.pulumi.com/logos/tech/logo-ts.png" height=38 />     | [TypeScript](./sdk/nodejs) | Stable  | Node.js 14+  |
+| <img src="https://www.pulumi.com/logos/tech/logo-python.svg" height=38 /> | [Python](./sdk/python)     | Stable  | Python 3.6+ |
 | <img src="https://www.pulumi.com/logos/tech/logo-golang.png" height=38 /> | [Go](./sdk/go)             | Stable  | Go 1.14+   |
```
Comment (susanev) [inline README.md:193]:
```suggestion
| <img src="https://www.pulumi.com/logos/tech/logo-js.png" height=38 />     | [JavaScript](https://www.pulumi.com/docs/intro/languages/javascript/) | Stable  | Node.js 14+  |
| <img src="https://www.pulumi.com/logos/tech/logo-ts.png" height=38 />     | [TypeScript](https://www.pulumi.com/docs/intro/languages/javascript/) | Stable  | Node.js 14+  |
| <img src="https://www.pulumi.com/logos/tech/logo-python.svg" height=38 /> | [Python](https://www.pulumi.com/docs/intro/languages/python/)     | Stable  | Python 3.6+ |
| <img src="https://www.pulumi.com/logos/tech/logo-golang.png" height=38 /> | [Go](https://www.pulumi.com/docs/intro/languages/go/)             | Stable  | Go 1.14+   |
```

File: README.md [hunk 3]
```diff
@@ -187,11 +187,13 @@ The Pulumi CLI v1 and v2 are no longer supported. If you are not yet running v3,
...
-| <img src="https://www.pulumi.com/logos/tech/dotnet.png" height=38 />      | [.NET (C#/F#/VB.NET)](./sdk/dotnet)     | Stable  | .NET Core 3.1+  |
+| <img src="https://www.pulumi.com/logos/tech/dot-net.svg" height=38 />      | [.NET (C#/F#/VB.NET)](./sdk/dotnet)     | Stable  | .NET Core 3.1+  |
+| <img src="https://www.pulumi.com/logos/tech/java.svg" height=38 />      | [Java](https://github.com/pulumi/pulumi-java/tree/main/sdk)     | Public Preview  | JDK 11+  |
+| <img src="https://www.pulumi.com/logos/tech/yaml.svg" height=38 />      | [YAML](https://www.pulumi.com/docs/reference/yaml/)     | Public Preview  | n/a  |
```
Comment (susanev) [inline README.md:196]:
```suggestion
| <img src="https://www.pulumi.com/logos/tech/dot-net.svg" height=38 />      | [.NET (C#/F#/VB.NET)](https://www.pulumi.com/docs/intro/languages/dotnet/)     | Stable  | .NET Core 3.1+  |
| <img src="https://www.pulumi.com/logos/tech/java.svg" height=38 />      | [Java](https://www.pulumi.com/docs/intro/languages/java/)     | Public Preview  | JDK 11+  |
| <img src="https://www.pulumi.com/logos/tech/yaml.svg" height=38 />      | [YAML](https://www.pulumi.com/docs/intro/languages/yaml/)     | Public Preview  | n/a  |
```

**Pull request merged**